.changeset/config.json

```diff
@@ -0,0 +1,100 @@
+      "@commercetools-uikit/creatable-select-input",
+      "@commercetools-uikit/data-table",
+      "@commercetools-uikit/date-field",
+      "@commercetools-uikit/date-input",
+      "@commercetools-uikit/date-range-field",
+      "@commercetools-uikit/date-range-input",
+      "@commercetools-uikit/date-time-field",
+      "@commercetools-uikit/date-time-input",
+      "@commercetools-uikit/design-system",
+      "@commercetools-uikit/field-errors",
+      "@commercetools-uikit/field-label",
+      "@commercetools-uikit/fields",
+      "@commercetools-uikit/flat-button",
+      "@commercetools-uikit/grid",
+      "@commercetools-uikit/hooks",
+      "@commercetools-uikit/i18n",
+      "@commercetools-uikit/icon-button",
+      "@commercetools-uikit/icons",
+      "@commercetools-uikit/inline",
+      "@commercetools-uikit/inputs",
+      "@commercetools-uikit/inset",
+      "@commercetools-uikit/inset-squish",
+      "@commercetools-uikit/label",
+      "@commercetools-uikit/link",
+      "@commercetools-uikit/link-button",
+      "@commercetools-uikit/loading-spinner",
+      "@commercetools-uikit/localized-money-input",
+      "@commercetools-uikit/localized-multiline-text-field",
+      "@commercetools-uikit/localized-multiline-text-input",
+      "@commercetools-uikit/localized-rich-text-input",
+      "@commercetools-uikit/localized-text-field",
+      "@commercetools-uikit/localized-text-input",
+      "@commercetools-uikit/localized-utils",
+      "@commercetools-uikit/messages",
+      "@commercetools-uikit/money-field",
+      "@commercetools-uikit/money-input",
+      "@commercetools-uikit/multiline-text-field",
+      "@commercetools-uikit/multiline-text-input",
+      "@commercetools-uikit/notifications",
+      "@commercetools-uikit/number-field",
+      "@commercetools-uikit/number-input",
+      "@commercetools-uikit/password-field",
+      "@commercetools-uikit/password-input",
+      "@commercetools-uikit/primary-action-dropdown",
+      "@commercetools-uikit/primary-button",
+      "@commercetools-uikit/radio-field",
+      "@commercetools-uikit/radio-input",
+      "@commercetools-uikit/rich-text-input",
+      "@commercetools-uikit/secondary-button",
+      "@commercetools-uikit/secondary-icon-button",
+      "@commercetools-uikit/select-field",
+      "@commercetools-uikit/select-input",
+      "@commercetools-uikit/select-utils",
+      "@commercetools-uikit/spacings",
+      "@commercetools-uikit/stack",
+      "@commercetools-uikit/stamp",
+      "@commercetools-uikit/table",
+      "@commercetools-uikit/tag",
+      "@commercetools-uikit/text",
+      "@commercetools-uikit/text-field",
+      "@commercetools-uikit/text-input",
+      "@commercetools-uikit/time-field",
+      "@commercetools-uikit/time-input",
+      "@commercetools-uikit/toggle-input",
+      "@commercetools-uikit/tooltip",
+      "@commercetools-uikit/ui-kit",
+      "@commercetools-uikit/utils"
+    ]
+  ],
```

Comment:
This should keep more or less the same behavior of lerna "locked" mode, where all packages stay more or less at the same version.

If we don't want this behavior we can remove the list.